appengine/analytics/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>servlet-api</artifactId>
-      <version>2.5</version>
+      <version>3.0-alpha-1</version>
```

Comment:
@lesv I imagine we want to exclude alphas for the servlet API.

Maybe event exclude the servlet API altogether from updates?